python-avocado.spec

```diff
@@ -38,6 +38,7 @@ Source0: https://github.com/avocado-framework/%{srcname}/archive/%{version}.tar.
 %else
 Source0: https://github.com/avocado-framework/%{srcname}/archive/%{commit}.tar.gz#/%{gittar}
 %endif
+Patch0: 0001-selftests-unit-test_utils_cpu.py-skip-tests-on-older.patch
```

Comment:
OK

Reply:
I understand this is to keep the history, but where is the file located? I know you can't re-write the history, but without the file being available the history doesn't make sense to me (as rpm-beginner)